Analysis/include/Coda3EventDecoder.h

```diff
@@ -0,0 +1,112 @@
+#ifndef	CODA3EVENTDECODER_H
+#define CODA3EVENTDECODER_H
+
+#include "VEventDecoder.h"
+#include "Rtypes.h"
+
+#include <vector>
+
+class Coda3EventDecoder : public VEventDecoder
+{
+public:
+		Coda3EventDecoder() : fControlEventFlag(kFALSE), TSROCNumber(0) { }
```

Comment:
fControlEventFlag -- 
Coda 2 handles Control Events a bit differently (ie less rigidly) than Coda 3.

-  In Coda 2, if an event has the fBankIDNum = 0xCC but it is not a physics event, we pass it through ProcessControlEvent *just in case* its a control event. If it is not, the event passes harmlessly through.
- In Coda 3, the control events are checked for at the same time we check for physics events (via a switch statement). fControlEventFlag allows us to indicate an event is a control event and handle it later on. 

TSROCNumber -- currently set to 0 and never modified
Coda 3 creates a trigger bank with TI information (time, eventnumber, trigger bits, etc). For single TIs and single block_level, this is no issue as we just take the one TI value and move on. BUT! If we have several TIs or mutliblock, the we will get multiple TI bank information (time, trigger_bits, etc). We might want to select the TS data as the "data". 

TSROCNumber is used as the index of the TS; Podd uses a crate map to define this. We might want to open this up to a the user to set (via command line option?) down the road. The exact logic is not yet explored.